.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on: [push]
```

Comment:
Don't we need the branch filter specified as well? I figured we would want it to be only on pushes to `main`

Reply:
Yep that’s an oversight

Reply:
It also releases canary builds on push to PR